committee-steering/governance/annual-reports.md

```diff
@@ -0,0 +1,125 @@
+# Kubernetes Community Group Annual Reports
+
+This document outlines the bootstrap process for SIG, WG, and UG Chairs and Tech
+Leads and Committees to establish a community group health check with Steering.
+All policy updates will be in their respective [SIG], [WG], and [UG]
+ governance docs as well as the general [governance] guidance.
+
+## Goals
+- Paint a complete project health picture for all of our community groups
+- Create a feedback loop between Chairs, the community groups, and the Steering
+Committee to move the project forward  
+- Encourage dialogue about the wellbeing of the projects contributors and offer
+suggested guidance and coaching
+- Promote healthy, active, engaged community groups  
+- Understand and have context before issues arise and celebrate wins where they  
+should be highlighted  
+- Help reshape project priorities at a high level
+
+## Reporting Process  
+
+Chairs are responsible for compiling a yearly public report but may be
+completed with the help of members of that group. Chairs are responsible for
+ensuring the reports are complete, accurate, and submitted to the Steering
+Committee.
+
+### 2020:
+Due to playing catch up with 40 community groups, we will have a slightly
+modified timeline bootstrap process this year and then will follow the process
+outlined in 2021. This means that Chairs will be reporting twice in n months and
+not a year. A yearly cadence will be followed post 2021.
+
+TODO outline time line for 2020  
+
+### 2021 and on:
+1. Chairs create one report and send it to steering-private@kubernetes.io for
+review; reports are due at the end of February every year and can pick a
+collaboration doc tool of their choice (google docs, hackmd, etc)
+2. Steering Committee will assign one member to each community group before the
+reports are submitted so there is an established point of contact for any help or
+guidance needed
+3. The March SIG Chairs and Tech Leads meeting will be used as follow up and
+report retrospectives
+4. Steering will schedule private meetings to follow up on anything denoted as
+a private matter on the report; reports are not made public until all private
+comments have been addressed
+5. Once reviewed and follow ups are completed, Steering will ask the chair(s)
+of the group to submit a pull request to that groups documentation in the
+community repo.
+6. The Steering Committee liaison will work directly with groups that have follow
+up items and update Steering during regular monthly meetings
+
+### Tips for Chairs:  
+- SIG Chairs are responsible for collecting and reporting information for
+sponsored Working Groups.    
```

Comment:
In the first line of the doc, we say:

> This document outlines the bootstrap process for SIG, WG, and UG Chairs and Tech
Leads and Committees to establish a community group health check with Steering.

My understanding was that even WG chairs would fill this report. Is the intention that SIGs will fill the reports on "behalf" of their WGs? If yes, two points around it:

1. We don't really have a good WG -> SIG mapping. One WG might related to many SIGs. For example, if WG `A` relates to SIG `X`, `Y`, `Z`, are the three SIGs required to include info about WG `A` in their report?

2. We might get more specific feedback if we poll the WGs directly... :thinking:

Reply:
i think having a WG provide a report is preferred because many times multiple SIGs may "sponsor" an activity, but members within the SIG do not always attend while the WG work product is established.  To some measure, the WG report is important feedback to the sponsoring SIG on its status.

Reply:
ack - changed